statsmodels/stats/_delta_method.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+"""
+Author: Josef Perktold
+License: BSD-3
+
+"""
+
+from __future__ import print_function
+import numpy as np
+from scipy import stats
+
+class NonlinearDeltaCov(object):
+    '''Asymptotic covariance by Deltamethod
+
+    the function is designed for 2d array, with rows equal to
+    the number of equations and columns equal to the number
```

Comment:
it's a class now not a function

Can parts be moved or shared with ContrastResults, like the model Results.t_test results?